integr8ly/grafana-operator
# Support env from secret

## Description

Support for env vars provided from secrets or config maps using `envFrom`. Based #176 

This also solves the problem of external admin credentials overridden by the operator. The new flag `spec.deployment.skipCreateAdminAccount` disables the operator from creating an admin account.

ping @kfyharukz

## Type of change



## Checklist


## Verification steps

1. Deploy Grafana and provide env vars including `GF_SECURITY_ADMIN_USER` and `GF_SECURITY_ADMIN_PASSWORD` from a secret.
3. Make sure that `spec.deployment.skipCreateAdminAccount` is set to `true`.
4. Make sure the provided login credentials work.
5. Open a terminal inside the Grafana container and make sure the login credentials work.
